avro2py/utils.py

```diff
@@ -51,6 +51,13 @@ def _safe_convert_namedtuple(v: Union[NamedTuple, Any]) -> Union[AvroDictType, A
     return v
 
 
+@_CONVERTERS_TO_AVRO.append
+def _safe_convert_list(v: Union[List, Any]) -> Union[AvroDictType, Any]:
+    if isinstance(v, list) and hasattr(v[0], '_asdict'):
```

Comment:
Won't this fail with `IndexError` if the `v` is empty? I would either add a new test or modify https://github.com/hchasestevens/avro2py/pull/11/files#diff-33c13e0b177bacd2f02e29bcb8aea5b49e7ce34901fd8f41fefb65defba1bd33R28-R37 to be hypothesis-generated (i.e. take in `st.lists(st.builds(WallyworldAdItem))`, which would have caught this.

Reply:
It does fail for empty list. Added another test, as suggested.